components/server/src/env.ts

```diff
@@ -89,8 +89,8 @@ export class Env extends AbstractComponentEnv {
     readonly authProviderConfigs = this.parseAuthProviderParamss();
 
     protected parseAuthProviderParamss(): AuthProviderParams[] {
-        const envVar = getEnvVar('AUTH_PROVIDERS_CONFIG');
-        return parseAuthProviderParamsFromEnv(JSON.parse(envVar));
+        // const envVar = getEnvVar('AUTH_PROVIDERS_CONFIG');
+        return [];
```

Comment:
to simulate a SH installation

Reply:
We should remove the content in the `values.yaml`.

Reply:
let's do that independently, ok?

Reply:
K, did already.

Reply:
@geropl what does this mean?

> K, did already.